bundle-size/test/api.test.js

```diff
@@ -371,6 +380,83 @@ describe('bundle-size api', () => {
       });
     });
 
+    test.each([
+      [
+        'draft = true',
+        pullRequest => {
+          pullRequest.draft = true;
+        },
+      ],
+      [
+        'title has "WIP" in it',
+        pullRequest => {
+          pullRequest.title = `[WIP] ${pullRequest.title}`;
+        },
+      ],
+    ])(
+      'update a check on bundle-size report with no approvers (report/base = 12.34KB/12.00KB) for PR with %s',
```

Comment:
I'm having trouble following what this is testing; it seems like a lot given what's actually changing, and the test description doesn't mention that that this is testing whether or not reviewers are added based on the title/status of the PR. Maybe add a standalone function `shouldAddReviewers` or something like that which can be more easily unit tested?

Reply:
These tests honestly need a serious overhaul. I changed the name of the test to be more clear as to what it checks for, but I'll defer the major overhaul to [probably never 🙃]

Reply:
I didn't notice the %s in the test name, so that makes it a bit clearer. Thanks!